internal/test_helpers/fixtures/printing_matches/success

```diff
@@ -0,0 +1,71 @@
+[33m[tester::#KU5] [0m[92m✓ Stdout contains 1 expected line in order[0m
+[33m[tester::#KU5] [0m[94m$ echo -ne "prefix_apple" | ./your_grep.sh -E '^apple'[0m
+[33m[tester::#KU5] [0m[92m✓ Received exit code 1.[0m
+[33m[tester::#KU5] [0m[92m✓ Found line "" (empty)[0m
+[33m[tester::#KU5] [0m[92m✓ Stdout contains 1 expected line in order[0m
+[33m[tester::#KU5] [0m[94m$ echo -ne "cat" | ./your_grep.sh -E 'ca+t'[0m
+[33m[your_program] [0mcat
+[33m[tester::#KU5] [0m[92m✓ Received exit code 0.[0m
+[33m[tester::#KU5] [0m[92m✓ Found line "cat"[0m
+[33m[tester::#KU5] [0m[92m✓ Stdout contains 1 expected line in order[0m
+[33m[tester::#KU5] [0m[94m$ echo -ne "ptest" | ./your_grep.sh -E '[pineapple]'[0m
+[33m[your_program] [0mptest
+[33m[tester::#KU5] [0m[92m✓ Received exit code 0.[0m
+[33m[tester::#KU5] [0m[92m✓ Found line "ptest"[0m
+[33m[tester::#KU5] [0m[92m✓ Stdout contains 1 expected line in order[0m
+[33m[tester::#KU5] [0m[92mTest passed.[0m
+
+[33m[tester::#PZ6] [0m[94mRunning tests for Stage #PZ6 (pz6)[0m
+[33m[tester::#PZ6] [0m[94m$ echo -ne "apple\nbanana123\ncherry\ndog456\nelephant" | ./your_grep.sh -E '\d'[0m
+[33m[your_program] [0mbanana123
+[33m[your_program] [0mdog456
+[33m[tester::#PZ6] [0m[92m✓ Received exit code 0.[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "banana123"[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "dog456"[0m
+[33m[tester::#PZ6] [0m[92m✓ Stdout contains 2 expected lines in order[0m
+[33m[tester::#PZ6] [0m[94m$ echo -ne "apple\nbanana\ncherry\ndog" | ./your_grep.sh -E '\d'[0m
+[33m[tester::#PZ6] [0m[92m✓ Received exit code 1.[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "" (empty)[0m
+[33m[tester::#PZ6] [0m[92m✓ Stdout contains 1 expected line in order[0m
+[33m[tester::#PZ6] [0m[94m$ echo -ne "grape\n!@#$\napple\n+++\ntest123" | ./your_grep.sh -E '\w+'[0m
+[33m[your_program] [0mgrape
+[33m[your_program] [0mapple
+[33m[your_program] [0mtest123
+[33m[tester::#PZ6] [0m[92m✓ Received exit code 0.[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "grape"[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "apple"[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "test123"[0m
+[33m[tester::#PZ6] [0m[92m✓ Stdout contains 3 expected lines in order[0m
+[33m[tester::#PZ6] [0m[94m$ echo -ne "france\ntiger\nitaly\nfox\nspain" | ./your_grep.sh -E '(tiger|fox)'[0m
+[33m[your_program] [0mtiger
+[33m[your_program] [0mfox
+[33m[tester::#PZ6] [0m[92m✓ Received exit code 0.[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "tiger"[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "fox"[0m
+[33m[tester::#PZ6] [0m[92m✓ Stdout contains 2 expected lines in order[0m
+[33m[tester::#PZ6] [0m[94m$ echo -ne "New York\nWashington D.C.\nAustin\nLos Angeles" | ./your_grep.sh -E '(tiger|fox)'[0m
+[33m[tester::#PZ6] [0m[92m✓ Received exit code 1.[0m
+[33m[tester::#PZ6] [0m[92m✓ Found line "" (empty)[0m
```

Comment:
@UdeshyaDhungana this is a very indirect log - we should instead say something like "✔︎ No output found".